examples/sddmm_example.py

```diff
@@ -6,7 +6,7 @@
 import scipy.sparse as sps
 
 LEN = 10000
-DENSITY = 0.0001
+DENSITY = 0.00001
```

Comment:
Why do we need this change?

Reply:
This example runs a bit faster now - SDDMM takes the longest time out of all examples. It also shows Finch speedup even better.